types/webpack/index.d.ts

```diff
@@ -1622,7 +1665,22 @@ declare namespace webpack {
     }
 
     class DefinePlugin extends Plugin {
-        constructor(definitions: { [key: string]: any });
+        constructor(definitions: { [key: string]: DefinePlugin.CodeValuePrimitive});
+        static runtimeValue(
+            fn: ({ module }: { module: compilation.Module }) => DefinePlugin.CodeValuePrimitive,
+            fileDependencies?: string[]
+        ): DefinePlugin.RuntimeValue;
+    }
+
+    namespace DefinePlugin {
+        class RuntimeValue {
+            constructor(
+                fn: ({ module }: { module: compilation.Module }) => CodeValuePrimitive,
+                fileDependencies?: string[]
```

Comment:
This should be

```ts
fileDependencies?: true | string[]
```

https://github.com/webpack/webpack/blob/89c752411e9bd803401acf1b8402559dae4c6eea/lib/DefinePlugin.js#L30